src/ParagraphsFeatures.php

```diff
@@ -109,6 +115,17 @@ public static function getThirdPartyForm(WidgetInterface $plugin, $field_name) {
       ],
     ];
 
+    // Only show the drag & drop feature if we can find the sortable library.
+    $library_discovery = \Drupal::service('library.discovery');
+    $library = $library_discovery->getLibraryByName('paragraphs', 'paragraphs-dragdrop');
+
+    $elements['show_drag_and_drop'] = [
+      '#type' => 'checkbox',
+      '#title' => t('Show drag & drop button'),
+      '#default_value' => $plugin->getThirdPartySetting('paragraphs_features', 'show_drag_and_drop'),
```

Comment:
I think this should default to true as well, otherwise it unsuspected to change behavior by submitting the form without changing the value.